### PR TITLE
Transmissions bend like crazy

### LINKED ISSUE #952: Create random bendings on transmissions

### PR DESCRIPTION
**Pull request by rilifon**
Closes #952

### CODE REVIEW

File: game/view/transmission.lua [hunk 1]
```diff
@@ -29,6 +32,16 @@ function Transmission:init(origin, target, color, duration)
...
 
+  --Create bendings
+  self.bending_number = bending_number or 6
+  self.bending_offsets = {}
+  for i = 1, self.bending_number do
+    self.bending_offsets[i] = RANDOM.safeGenerate(_MIN_BEND_OFFSET, _MAX_BEND_OFFSET)
+  end
+  self.bending_angles = {}
+  for i = 1, self.bending_number do
+    self.bending_angles[i] = RANDOM.safeGenerate(_BEND_ANGLE, _BEND_ANGLE)
```
Comment (OrenjiAkira) [inline game/view/transmission.lua]: why use a random for an empty interval?

File: game/view/transmission.lua [hunk 2]
```diff
@@ -46,23 +59,39 @@ end
...
+    local pos = ori+norm*dist*i
+    pos = pos + norm:perpendicular():rotate(angle)*off*sign
+    table.insert(points,pos.x)
+    table.insert(points,pos.y)
+    sign = sign*-1
+  end
+  --Insert target position
+  table.insert(points, tar.x)
+  table.insert(points, tar.y)
+  g.line(unpack(points))
```
Comment (OrenjiAkira) [inline game/view/transmission.lua]: não é bom usar unpack para muitos atributos pq tem um limite. o graphics.line aceita uma tabela de pontos.

**Comment by OrenjiAkira** [comment]
Also remember to fix them conflicts please.

File: game/view/transmission.lua (hunk 1 above)
Comment (rilifon) [reply game/view/transmission.lua]: That was a mistake >< It was supposed to be -_BEND_ANGLE to _BEND_ANGLE

File: game/view/transmission.lua (hunk 2 above)
Comment (rilifon) [reply game/view/transmission.lua]: Uhh forgot about that

**Comment by rilifon** [comment]
@OrenjiAkira Fixed things you mentioned and rebased with dev

**Pull request merged**